project_euler/problem_074/sol2.py

```diff
@@ -86,25 +94,27 @@ def solution(chain_length: int = 60, number_limit: int = 1000000) -> int:
     for i in range(1, number_limit + 1):
 
         # The temporary list will contain the elements of the chain
-        chain_list = [i]
+        chain_set = {i}
+        len_chain_set = 1
+        last_chain_element = i
 
         # The new element of the chain
-        new_chain_element = factorial_sum(chain_list[-1])
+        new_chain_element = factorial_sum(last_chain_element)
 
         """ Stop computing the chain when you find a repeating item
```

Comment:
Also, if possible please convert these multiline strings into comments wherever they are being placed.

Reply:
I am not touching anything that changes this, it is still a valid comment.
And I don't feel multi-line strings VS multiline comments is really affecting code quality/readability and would prefer not to change that bit

Reply:
I am not asking you to change the semantics of the comment but the structure of it. The way it is written is not considered good practice and we try to maintain consistency throughout the repository. I am just suggesting to convert it to a normal comment instead of a string. Multiline strings are used mostly, if not only, for function docstring and module-level docstring.